packages/blocks/src/page-block/keyboard/keyboard-manager.ts

```diff
@@ -18,6 +19,14 @@ export class PageKeyboardManager {
             this._page.redo();
           }
         },
+        'Control-y': ctx => {
+          if (IS_WINDOWS) {
```

Comment:
We prefer the `if (!IS_WINDOWS) return` guard pattern, this lowers the code nesting and makes condition more straightforward to understand.

Reply:
I just thought about whether to write it like this haha, I will change it now